src/app/components/home/home.component.scss

```diff
@@ -178,22 +188,37 @@
   }
 }
 
-.header-nav {
-  nav {
-    display: flex;
-    flex-direction: row;
-    .nav_links {
-      color: white;
-      margin: 0px 20px;
-      a {
-        margin: 0;
-      }
-    }
+.header-nav{
+  .bar-toggle{
+    display:none;
+    color:#fff;
   }
-
+  font-size: 18px;
+  display: grid;
+  grid-template-columns: 3fr auto auto;
+  padding: 10px 20px;
+  text-align: left;
+  background-color: $vvdarkblue;
+  align-items: center;
+  .nav_links {
+      margin-left: 20px;
+      margin-right: 20px;
+      color: #ffffff;
+      text-decoration: none;
+      font-weight: 100;
+      &:hover {
+          text-decoration: underline;
+          color: $vvyellow;
+        }
+        &:active {
+          
```

Comment:
Not sure why there's an extra line here.

Reply:
Fixed